src/main/java/nextstep/subway/line/dto/LineRequest.java

```diff
@@ -2,7 +2,10 @@
 
 import nextstep.subway.line.domain.Line;
 
+import java.util.Objects;
+
 public class LineRequest {
+    private static final LineRequest EMPYTY = new LineRequest("", "");
     private String name;
     private String color;
```

Comment:
Bean validation 사용을 고려해보시면 어떨까요?

```suggestion
    @NotBlank
    private String name;
    @NotBlank
    private String color;
```